mappings/net/minecraft/world/biome/layer/HillsLayer.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS chb net/minecraft/world/biome/layer/HillsLayer
+	FIELD A snowTaigaHills I
```

Comment:
Are these constants? Potentially should be `UPPER_SNAKE_CASE`.

Reply:
They're constants.